src/cstd.h

```diff
@@ -304,6 +304,46 @@ stdnone_of(void *arr, size_t stride, size_t len, int (boolfunc)(const void *))
   return 1;
 }
 
+// Swaps element a with element b
+// complains if either is null for some reason
+#define __STDSWAP(type)                     \
+        void stdswap_##type(type* _val1, type* _val2) \
+        {                                             \
+          if (_val1 == NULL || _val2 == NULL)         \
+            __STD_PANIC("Error: null pointer detected in stdswap_%s", #type); \
```

Comment:
You don't have to put "Error" here, __STD_PANIC already puts a [PANIC] which is the same as an error.